plugins/chai-plugin-screenshot/src/index.js

```diff
@@ -102,41 +99,41 @@ const plugin = {
         : `${id}.png`;
       const basePath = isTakeImageOf ? meta.artifactsPath : artifactsPath;
 
-      const baselinePath = path.resolve(
-        basePath,
-        'baseline',
-        folder,
-        imageName,
-      );
-      const regressionPath = path.resolve(
+      const baseline = path.resolve(basePath, 'baseline', folder, imageName);
+      const regression = path.resolve(
         basePath,
         'regression',
         folder,
         imageName,
       );
-      const diffPath = path.resolve(basePath, 'diff', folder, imageName);
-
+      const diff = path.resolve(basePath, 'diff', folder, imageName);
+      const expected = JSON.stringify({
+        baseline,
+        diff,
+        regression,
+      });
+      const actual = {};
```

Comment:
actual is suppose to be an empty object?

Reply:
I just looked at the aw 4 version but we might need some investigation